src/lib/games/queries.ts

```diff
@@ -233,8 +233,30 @@ export async function getGameResultsData(tx: DBOrTransation, gameId: string) {
     where: cmp.eq(schema.questions.id, game.question_id),
   })
 
+  const playersWithFinalCode = await Promise.all(
```

Comment:
now what we're not adding a `limit`, we can do this in 1 query and group by `player_game_session_id` w/ js